### DISCUSSION TITLE
unreadable system font

### DISCUSSION THREAD

**Discussion by Ermudu54** [elementary/os#624] [Q&A] [answered]
Hi,
I'm having a weird problem: The system font has become unreadable. See screenshot.
Each letter is replaced by the same shape that imitates handwriting.
![Capture d'écran du 2022-08-05 18 21 54](https://user-images.githubusercontent.com/110684133/183119818-51daa84f-6bd4-4cd8-8ed7-3148b8b0e190.jpeg)
OS Elementary (I don't know witch one, and can't read it now !)
Did I accidentally change a setting? (I fell asleep in front of my screen yesterday)
Is it malware?
Do you have an explanation?
Thank's
![Capture d'écran du 2022-08-05 18 17 39](https://user-images.githubusercontent.com/110684133/183119192-f846d7b0-96cb-4bd3-a75d-cf7e3038a34b.jpeg)

**Comment by jeremypw** [comment]
What you are seeing is the special font used to obscure text in e.g. screenshots. Not obvious how this could accidentally become the desktop font though.  You could try using `dconf-editor` or the command line to look at the `org/gnome/desktop/interface/font-name` setting and reset it to the default.

**Comment by jeremypw** [comment, marked as answer]
The command to reset the system font is `gsettings reset org.gnome.desktop.interface font-name`

**Reply by Ermudu54** [reply]
Oh, thank you very much !!!! 
With the dconf editor I was unable to read something ... due to my problem ;-).
Your command save me !  Je vous remercie beaucoup.

**Reply by Ermudu54** [reply]
I don't know how the problem came, but yesterday I took screenshots.

**Reply by jeremypw** [reply]
Glad to help. If it happens again then consider raising a bug report (especially if it is reproducible).

**Comment by aeonoea** [comment]
I am having the same problem on a vanilla install with the latest version. How is that still an issue two years later...?

Resetting didn't work of course, because I haven't changed anything yet.

**Reply by tintou** [reply]
You might have used the screenshot tool with text obfuscation which changes the font, `gsettings reset org.gnome.desktop.interface font-name`

**Reply by aeonoea** [reply]
I also thought that but opening the screenshot tool reveals that obfuscation is turned off. Also resetting doesn't work as outlined above unfortuntely.

Btw I opened a new issue [here](https://github.com/elementary/stylesheet/issues/1307)
